corehq/apps/export/views.py

```diff
@@ -488,8 +488,11 @@ def download_export_form(self):
         raise NotImplementedError("You must implement download_export_form.")
 
     @staticmethod
-    def get_export_schema(export_id):
-        return get_document_or_404_lite(SavedExportSchema, export_id)
+    def get_export_schema(domain, export_id):
+        doc = get_document_or_404_lite(SavedExportSchema, export_id)
```

Comment:
https://github.com/dimagi/commcare-hq/pull/10030 That checks domains and for some reason SavedExportSchema has null as domain. That's why I'm checking the index below